templates/cf.yml

```diff
@@ -618,6 +639,8 @@ properties:
     tls:
       ca_cert: ~
     etcd:
+      ca_cert: (( .properties.etcd.ca_cert ))
+      require_ssl: (( .properties.etcd.require_ssl ))
```

Comment:
I see.  The clients don't care about cluster, but they do need to know how to talk to the server, which is what "machines" is for.  Now this tricky.  In non-tls mode, the "machines" should be the list of IPs.  But in TLS mode, it should be the single DNS name of the secure etcd cluster, in this case "cf-etcd.service.cf.internal".  See e.g. https://github.com/cloudfoundry-incubator/diego-release/blob/develop/jobs/bbs/spec#L59-L61.  I'm not sure what the right thing to do here for spiff.